src/NodeContainer/ParsedNodesByType.php

```diff
@@ -484,6 +491,21 @@ private function addClassConstant(ClassConst $classConst): void
     }
 
     private function addClassConstantFetch(ClassConstFetch $classConstFetch): void
+    {
+        $this->classConstantFetches[] = $classConstFetch;
+    }
+
+    private function processClassConstantFetches(): void
+    {
+        if ($this->classConstantFetches) {
+            foreach ($this->classConstantFetches as $rawClassConstantFetch) {
+                $this->processClassConstantFetch($rawClassConstantFetch);
+            }
+            $this->classConstantFetches = [];
+        }
+    }
+
+    private function processClassConstantFetch(ClassConstFetch $classConstFetch): void
```

Comment:
What does "process" stand for? Better naming would help to understand

Reply:
It's processing the `ClassConstFetch` items that have been collected during `collect`, resolving them to their respective classes. Everything that was previously done in the `addClassConstantFetch` call. Maybe "resolve" or "evaluate" is the better name?